File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L3/04/Kasus4.ts
/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
function Konversi(n: number): void {
    let miles: number = 1;
    while (miles <= n) {
        console.log(`${miles}\t\t${miles * 1.609}`);
        miles++;
    }
}

console.log("Miles\t\tKilometers");
console.log("-------------------------------");
// Use while loop
Konversi(10);

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L5/04/Kasus4.kt
package IR

/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
object Kasus4 {
    fun Konversi(miles: Int): Double {
        return miles * 1.609
    }

    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")
        println("-------------------------------")
        // Use for loop
        for (miles in 1..10) {
            println("$miles\t\t${Konversi(miles)}")
        }
    }
}

